mixin/alerts/alerts.libsonnet

```diff
@@ -4,6 +4,6 @@
 (import 'sidecar.libsonnet') +
 (import 'store.libsonnet') +
 (import 'rule.libsonnet') +
-(import 'bucket_replicate.libsonnet') +
+(import 'bucket-replicate.libsonnet') +
 (import 'absent.libsonnet') +
 (import 'add_runbook_links.libsonnet')
```

Comment:
```suggestion
(import 'add-runbook-links.libsonnet')
```

Reply:
I haven't forgotten about this, I'm just stretched pretty thin right now. I tried to make the minimum amount of changes needed to get this working - there's a lot of files with snake_case names, are you suggesting they should all be renamed?

Reply:
I meant to focus on the files in the `mixin` directory. For sake of consistency. I don't see any other files left besides `add-runbool-links`. We can come back to linting part later.